File: Opencart_WebApp-Performance_Test_Scripts/SC003-OC_Place_an_order/data/action.js
//"'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
//" Script Title       : 
//"                      
//" Script Date        : Wed May 10 14:05:25 2017
//"                       
//"'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''

function Action()
{
	web.addCookie('SRCHUID=V=2&GUID=929B6FA041A640DFACCBCF7DC0980197; DOMAIN=www.bing.com');

	web.addCookie('SRCHD=AF=NOFORM; DOMAIN=www.bing.com');

	web.addCookie('SRCHUSR=DOB=20170405; DOMAIN=www.bing.com');

	web.url(
		{
			name : 'favicon.ico', 
			url : 'http://www.bing.com/favicon.ico', 
			resource : 1, 
			recContentType : 'image/x-icon', 
			referer : '', 
			snapshot : 't2.inf'
		}
	);

	web.addCookie('currency=EUR; DOMAIN=172.16.44.19');

	web.addCookie('__atuvc=9%7C15%2C15%7C16%2C0%7C17%2C1%7C18%2C3%7C19; DOMAIN=172.16.44.19');

	web.addCookie('language=en; DOMAIN=172.16.44.19');

	web.addCookie('__atuvs=5912eee00c3b7afe000; DOMAIN=172.16.44.19');

	web.regFind(
		{
			text : 'Your Store'
		}
	);

	web.url(
		{
			name : 'oc', 
			url : 'http://172.16.44.19/oc/', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : '', 
			snapshot : 't3.inf', 
			mode : 'HTML', 
			extraRes :  [
				{url : 'catalog/view/theme/default/image/button-previous.png'},
				{url : 'catalog/view/theme/default/image/button-next.png'},
				{url : 'catalog/view/theme/default/image/button-search.png'},
				{url : 'catalog/view/theme/default/image/menu.png'},
				{url : 'catalog/view/theme/default/image/loading.gif'},
				{url : 'catalog/view/theme/default/image/arrow-down.png'},
				{url : 'catalog/view/theme/default/image/background.png'},
				{url : 'catalog/view/theme/default/image/button.png'},
				{url : 'catalog/view/theme/default/image/bullets.png'},
				{url : 'catalog/view/theme/default/image/arrows.png'}
			]
		}
	);

	web.regFind(
		{
			text : 'Your Store'
		}
	);

	web.link(
		{
			name : 'Home', 
			text : 'Home', 
			snapshot : 't4.inf'
		}
	);

	web.regFind(
		{
			text : 'Account Login'
		}
	);

	lr.thinkTime(7);
	web.link(
		{
			name : 'login', 
			text : 'login', 
			snapshot : 't5.inf'
		}
	);

	web.regFind(
		{
			text : 'My Account'
		}
	);

	web.submitData(
		{
			name : 'index.php', 
			action : 'http://172.16.44.19/oc/index.php?route=account/login', 
			method : 'POST', 
			encType : 'multipart/form-data', 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=account/login', 
			snapshot : 't6.inf', 
			mode : 'HTML', 
			itemData :  [
				{name : 'email', value : 'test1@mail.com'},
				{name : 'password', value : 'test1'}
			]
		}
	);

	web.regFind(
		{
			text : 'Your Store'
		}
	);

	web.link(
		{
			name : 'Home_2', 
			text : 'Home', 
			ordinal : '1', 
			snapshot : 't7.inf'
		}
	);

	web.regFind(
		{
			text : 'Cameras'
		}
	);

	web.link(
		{
			name : 'Cameras', 
			text : 'Cameras', 
			snapshot : 't8.inf', 
			extraRes :  [
				{url : 'catalog/view/theme/default/image/add.png', referer : 'http://172.16.44.19/oc/index.php?route=product/category&path=33'}
			]
		}
	);

	web.setSocketsOption('SSL_VERSION', '2&3');

	web.addCookie('MC1=GUID=605b1210a00d984c98b65aa1bb3f1fe4&HASH=1012&LV=201612&V=4&LU=1482491531276; DOMAIN=iecvlist.microsoft.com');

	web.addCookie('A=I&I=AxUFAAAAAAA/CgAAw19EUVpKI2axcHutFCiCJA!!&V=4; DOMAIN=iecvlist.microsoft.com');

	web.addCookie('MSFPC=ID=605b1210a00d984c98b65aa1bb3f1fe4&CS=3&LV=201612&V=1; DOMAIN=iecvlist.microsoft.com');

	web.addCookie('SRCHD=AF=NOFORM; DOMAIN=iecvlist.microsoft.com');

	web.addCookie('SRCHUSR=DOB=20170102; DOMAIN=iecvlist.microsoft.com');

	web.addCookie('MUID=2966A66248FA6A192AC5AC364CFA6C73; DOMAIN=iecvlist.microsoft.com');

	lr.thinkTime(4);
	web.url(
		{
			name : 'iecompatviewlist.xml', 
			url : 'https://iecvlist.microsoft.com/IE11/1479242656000/iecompatviewlist.xml', 
			resource : 0, 
			recContentType : 'text/xml', 
			referer : '', 
			snapshot : 't9.inf', 
			mode : 'HTML'
		}
	);

	web.regFind(
		{
			text : 'Canon EOS 5D'
		}
	);

	web.url(
		{
			name : 'index.php_2', 
			url : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&product_id=30', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=product/category&path=33', 
			snapshot : 't10.inf', 
			mode : 'HTML', 
			extraRes :  [
				{url : 'catalog/view/theme/default/image/tab.png', referer : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&product_id=30'}
			]
		}
	);

	web.url(
		{
			name : 'index.php_3', 
			url : 'http://172.16.44.19/oc/index.php?route=product/product/review&product_id=30', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&product_id=30', 
			snapshot : 't11.inf', 
			mode : 'HTML'
		}
	);

	web.addCookie('uvc=9%7C15%2C15%7C16%2C0%7C17%2C1%7C18%2C3%7C19; DOMAIN=s7.addthis.com');

	web.addCookie('di2=aUj~t6Hq; DOMAIN=s7.addthis.com');

	web.addCookie('uid=58ecb5d16bf7095e; DOMAIN=s7.addthis.com');

	web.addCookie('vc=27; DOMAIN=s7.addthis.com');

	web.addCookie('loc=MDAwMDBFVU1EMDAyMjcwMjA4OTAwMDAwMDAwVg==; DOMAIN=s7.addthis.com');

	web.regFind(
		{
			text : 'AddThis Utility Frame'
		}
	);

	web.addCookie('__atuvc=9%7C15%2C15%7C16%2C0%7C17%2C1%7C18%2C4%7C19; DOMAIN=172.16.44.19');

	web.addCookie('__atuvs=5912eee00c3b7afe001; DOMAIN=172.16.44.19');

	web.url(
		{
			name : 'sh.f7354b64245ec78fa8b63827.html', 
			url : 'http://s7.addthis.com/static/sh.f7354b64245ec78fa8b63827.html', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&product_id=30', 
			snapshot : 't12.inf', 
			mode : 'HTML', 
			extraRes :  [
				{url : 'http://172.16.44.19/oc/catalog/view/javascript/jquery/colorbox/images/border.png', referer : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&product_id=30'},
				{url : 'http://172.16.44.19/oc/catalog/view/javascript/jquery/colorbox/images/loading_background.png', referer : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&product_id=30'},
				{url : 'http://172.16.44.19/oc/catalog/view/javascript/jquery/colorbox/images/controls.png', referer : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&product_id=30'},
				{url : 'http://172.16.44.19/oc/catalog/view/javascript/jquery/colorbox/images/loading.gif', referer : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&product_id=30'}
			]
		}
	);

	web.submitData(
		{
			name : 'index.php_4', 
			action : 'http://172.16.44.19/oc/index.php?route=checkout/cart/add', 
			method : 'POST', 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&product_id=30', 
			snapshot : 't13.inf', 
			mode : 'HTML', 
			itemData :  [
				{name : 'quantity', value : '1'},
				{name : 'product_id', value : '30'}
			],
			extraRes :  [
				{url : 'catalog/view/theme/default/image/close.png', referer : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&product_id=30'},
				{url : 'catalog/view/theme/default/image/success.png', referer : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&product_id=30'}
			]
		}
	);

	web.regFind(
		{
			text : 'Shopping Cart'
		}
	);

	web.addCookie('uvc=9%7C15%2C15%7C16%2C0%7C17%2C1%7C18%2C4%7C19; DOMAIN=m.addthis.com');

	web.addCookie('di2=aUj~t6Hq; DOMAIN=m.addthis.com');

	web.addCookie('uid=58ecb5d16bf7095e; DOMAIN=m.addthis.com');

	web.addCookie('vc=27; DOMAIN=m.addthis.com');

	web.addCookie('loc=MDAwMDBFVU1EMDAyMjcwMjA4OTAwMDAwMDAwVg==; DOMAIN=m.addthis.com');

	lr.thinkTime(4);
	web.url(
		{
			name : 'index.php_5', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/cart', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&product_id=30', 
			snapshot : 't14.inf', 
			mode : 'HTML', 
			extraRes :  [
				{url : 'http://m.addthis.com/live/red_lojson/100eng.json?sh=0&ph=1036&ivh=985&dt=3343&pdt=1294&ict=&pct=1&perf=widget%7C1294%7C4931%2Clojson%7C6419%7C0%2Csh%7C6423%7C4%2Csh%7C6425%7C188&cmenu=null&ppd=0&ppl=0&fbe=&xmv=0&xms=0&xmlc=0&jsfw=jquery&jsfwv=jquery-1.7.1&al=men&scr=0&scv=0&apiu=0&ba=1&sid=5912f3c9521c773d&rev=v7.13.0-wp&pub=&dp=172.16.44.19&fp=oc%2Findex.php%3Froute%3Dproduct%2Fproduct%26path%3D33%26product_id%3D30&pfm=1&icns=', referer : 'http://172.16.44.19/oc/index.php?route=product/product&path=33&' + 
		'product_id=30'}
			]
		}
	);

	web.url(
		{
			name : 'index.php_6', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/cart/country&country_id=140', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/cart', 
			snapshot : 't15.inf', 
			mode : 'HTML'
		}
	);

	web.regFind(
		{
			text : 'Checkout'
		}
	);

	lr.thinkTime(4);
	web.url(
		{
			name : 'index.php_7', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/cart', 
			snapshot : 't16.inf', 
			mode : 'HTML'
		}
	);

	web.url(
		{
			name : 'index.php_8', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/payment_address', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't17.inf', 
			mode : 'HTML'
		}
	);

	web.url(
		{
			name : 'index.php_9', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/checkout/country&country_id=222', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't18.inf', 
			mode : 'HTML'
		}
	);

	web.submitData(
		{
			name : 'index.php_10', 
			action : 'http://172.16.44.19/oc/index.php?route=checkout/payment_address/validate', 
			method : 'POST', 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't19.inf', 
			mode : 'HTML', 
			itemData :  [
				{name : 'payment_address', value : 'existing'},
				{name : 'address_id', value : '764'},
				{name : 'firstname', value : ''},
				{name : 'lastname', value : ''},
				{name : 'company', value : ''},
				{name : 'company_id', value : ''},
				{name : 'address_1', value : ''},
				{name : 'address_2', value : ''},
				{name : 'city', value : ''},
				{name : 'postcode', value : ''},
				{name : 'country_id', value : '222'},
				{name : 'zone_id', value : ''}
			]
		}
	);

	web.url(
		{
			name : 'index.php_11', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/payment_address', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't20.inf', 
			mode : 'HTML'
		}
	);

	web.url(
		{
			name : 'index.php_12', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/shipping_address', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't21.inf', 
			mode : 'HTML'
		}
	);

	web.url(
		{
			name : 'index.php_13', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/checkout/country&country_id=140', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't22.inf', 
			mode : 'HTML'
		}
	);

	web.url(
		{
			name : 'index.php_14', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/checkout/country&country_id=140', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't23.inf', 
			mode : 'HTML'
		}
	);

	web.submitData(
		{
			name : 'index.php_15', 
			action : 'http://172.16.44.19/oc/index.php?route=checkout/shipping_address/validate', 
			method : 'POST', 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't24.inf', 
			mode : 'HTML', 
			itemData :  [
				{name : 'shipping_address', value : 'existing'},
				{name : 'address_id', value : '764'},
				{name : 'firstname', value : ''},
				{name : 'lastname', value : ''},
				{name : 'company', value : ''},
				{name : 'address_1', value : ''},
				{name : 'address_2', value : ''},
				{name : 'city', value : ''},
				{name : 'postcode', value : 'md2020'},
				{name : 'country_id', value : '140'},
				{name : 'zone_id', value : '2182'}
			]
		}
	);

	web.url(
		{
			name : 'index.php_16', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/payment_address', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't25.inf', 
			mode : 'HTML'
		}
	);

	web.url(
		{
			name : 'index.php_17', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/shipping_method', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't26.inf', 
			mode : 'HTML'
		}
	);

	web.url(
		{
			name : 'index.php_18', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/checkout/country&country_id=140', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't27.inf', 
			mode : 'HTML'
		}
	);

	web.url(
		{
			name : 'index.php_19', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/shipping_address', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't28.inf', 
			mode : 'HTML'
		}
	);

	web.url(
		{
			name : 'index.php_20', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/checkout/country&country_id=140', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't29.inf', 
			mode : 'HTML'
		}
	);

	web.submitData(
		{
			name : 'index.php_21', 
			action : 'http://172.16.44.19/oc/index.php?route=checkout/shipping_method/validate', 
			method : 'POST', 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't30.inf', 
			mode : 'HTML', 
			itemData :  [
				{name : 'shipping_method', value : 'flat.flat'},
				{name : 'comment', value : ''}
			]
		}
	);

	web.url(
		{
			name : 'index.php_22', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/payment_method', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't31.inf', 
			mode : 'HTML'
		}
	);

	lr.thinkTime(5);
	web.submitData(
		{
			name : 'index.php_23', 
			action : 'http://172.16.44.19/oc/index.php?route=checkout/payment_method/validate', 
			method : 'POST', 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't32.inf', 
			mode : 'HTML', 
			itemData :  [
				{name : 'payment_method', value : 'cod'},
				{name : 'comment', value : ''},
				{name : 'agree', value : '1'}
			]
		}
	);

	web.url(
		{
			name : 'index.php_24', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/confirm', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't33.inf', 
			mode : 'HTML'
		}
	);

	web.url(
		{
			name : 'index.php_25', 
			url : 'http://172.16.44.19/oc/index.php?route=payment/cod/confirm', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't34.inf', 
			mode : 'HTML'
		}
	);

	web.regFind(
		{
			text : 'Your Order Has Been Processed!'
		}
	);

	lr.thinkTime(4);
	web.url(
		{
			name : 'index.php_26', 
			url : 'http://172.16.44.19/oc/index.php?route=checkout/success', 
			resource : 0, 
			recContentType : 'text/html', 
			referer : 'http://172.16.44.19/oc/index.php?route=checkout/checkout', 
			snapshot : 't35.inf', 
			mode : 'HTML'
		}
	);

	web.regFind(
		{
			text : 'Your Store'
		}
	);

	web.link(
		{
			name : 'Home_3', 
			text : 'Home', 
			ordinal : '1', 
			snapshot : 't36.inf'
		}
	);

	web.regFind(
		{
			text : 'Account Logout'
		}
	);

	web.link(
		{
			name : 'Logout', 
			text : 'Logout', 
			snapshot : 't37.inf'
		}
	);

	web.regFind(
		{
			text : 'Your Store'
		}
	);

	lr.thinkTime(5);
	web.link(
		{
			name : 'Home_4', 
			text : 'Home', 
			ordinal : '1', 
			snapshot : 't38.inf'
		}
	);

	return 0;
}

File: Opencart_WebApp-Performance_Test_Scripts/SC004-OC_Wish_list/Action.c
Action()
{

	web_add_cookie("currency=EUR; DOMAIN=172.16.44.19");
	web_add_cookie("__atuvc=9%7C15%2C15%7C16%2C0%7C17%2C1%7C18%2C4%7C19; DOMAIN=172.16.44.19");
	web_add_cookie("language=en; DOMAIN=172.16.44.19");
//-------------------------------------------------------------

	//Client opens home page
	
	web_reg_find("Text=Your Store", 
		LAST);

	web_url("Home_Page", 
		"URL=http://172.16.44.19/oc/", 
		"Resource=0", 
		"RecContentType=text/html", 
		"Referer=", 
		"Snapshot=t1.inf", 
		"Mode=HTML", 
		EXTRARES, 
		"Url=catalog/view/theme/default/image/button-search.png", ENDITEM, 
		"Url=catalog/view/theme/default/image/button-next.png", ENDITEM, 
		"Url=catalog/view/theme/default/image/arrow-down.png", ENDITEM, 
		"Url=catalog/view/theme/default/image/menu.png", ENDITEM, 
		"Url=catalog/view/theme/default/image/button-previous.png", ENDITEM, 
		"Url=catalog/view/theme/default/image/background.png", ENDITEM, 
		"Url=catalog/view/theme/default/image/loading.gif", ENDITEM, 
		"Url=catalog/view/theme/default/image/button.png", ENDITEM, 
		"Url=catalog/view/theme/default/image/bullets.png", ENDITEM, 
		"Url=http://www.bing.com/favicon.ico", "Referer=", ENDITEM, 
		"Url=catalog/view/theme/default/image/arrows.png", ENDITEM, 
		LAST);
//-------------------------------------------------------------

//Client opens login form
	
	web_reg_find("Text=Account Login", 
		LAST);

	web_link("Login_Form", 
		"Text=login", 
		"Snapshot=t2.inf", 
		LAST);
//-------------------------------------------------------------	
	
//Client fills and submits his credentials
	
       lr_start_transaction("Login");

	web_reg_find("Text=My Account", 
		LAST);

	web_submit_data("Login", 
		"Action=http://172.16.44.19/oc/index.php?route=account/login", 
		"Method=POST", 
		"EncType=multipart/form-data", 
		"RecContentType=text/html", 
		"Referer=http://172.16.44.19/oc/index.php?route=account/login", 
		"Snapshot=t3.inf", 
		"Mode=HTML", 
		ITEMDATA, 
		"Name=email", "Value=test1@mail.com", ENDITEM, 
		"Name=password", "Value=test1", ENDITEM, 
		LAST);

	lr_end_transaction("Login", LR_AUTO);
//-------------------------------------------------------------
	
//Client navigates to the home page
      
	web_reg_find("Text=Your Store", 
		LAST);

	web_link("Home_Page", 
		"Text=Home", 
		"Ordinal=1", 
		"Snapshot=t4.inf", 
		LAST);
//-------------------------------------------------------------
      
//Client chooses a product category
      
	web_reg_find("Text=Apple", 
		LAST);

	web_url("index.php_2", 
		"URL=http://172.16.44.19/oc/index.php?route=product/manufacturer/info&manufacturer_id=8", 
		"Resource=0", 
		"RecContentType=text/html", 
		"Referer=http://172.16.44.19/oc/index.php?route=common/home", 
		"Snapshot=t6.inf", 
		"Mode=HTML", 
		EXTRARES, 
		"Url=catalog/view/theme/default/image/add.png", "Referer=http://172.16.44.19/oc/index.php?route=product/manufacturer/info&manufacturer_id=8", ENDITEM, 
		LAST);
 //-------------------------------------------------------------     
      
//Client chooses a product

	web_reg_find("Text=iPod Nano", 
		LAST);

	lr_think_time(7);
	
	web_link("iPod Nano", 
		"Text=iPod Nano", 
		"Snapshot=t7.inf", 
		LAST);
//-------------------------------------------------------------
	
//Client adds chosen produc to wish list

	lr_think_time(4);
	
	web_submit_data("Add to wish list", 
		"Action=http://172.16.44.19/oc/index.php?route=account/wishlist/add", 
		"Method=POST", 
		"RecContentType=text/html", 
		"Referer=http://172.16.44.19/oc/index.php?route=product/product&manufacturer_id=8&product_id=36", 
		"Snapshot=t8.inf", 
		"Mode=HTML", 
		ITEMDATA, 
		"Name=product_id", "Value=36", ENDITEM, 
		EXTRARES, 
		"Url=catalog/view/theme/default/image/close.png", "Referer=http://172.16.44.19/oc/index.php?route=product/product&manufacturer_id=8&product_id=36", ENDITEM, 
		"Url=catalog/view/theme/default/image/success.png", "Referer=http://172.16.44.19/oc/index.php?route=product/product&manufacturer_id=8&product_id=36", ENDITEM, 
		"Url=catalog/view/theme/default/image/tab.png", "Referer=http://172.16.44.19/oc/index.php?route=product/product&manufacturer_id=8&product_id=36", ENDITEM, 
		LAST);
//-------------------------------------------------------------

//Client opens his wish list

	web_reg_find("Text=My Wish List", 
		LAST);
	
	lr_think_time(4);
	
	web_url("Wish List", 
		"URL=http://172.16.44.19/oc/index.php?route=account/wishlist", 
		"Resource=0", 
		"RecContentType=text/html", 
		"Referer=http://172.16.44.19/oc/index.php?route=product/product&manufacturer_id=8&product_id=36", 
		"Snapshot=t12.inf", 
		"Mode=HTML", 
		EXTRARES, 
		"Url=http://m.addthis.com/live/red_lojson/100eng.json?sh=0&ph=1388&ivh=985&dt=0&pdt=1317&ict=&pct=1&perf=widget%7C1317%7C4863%2Clojson%7C6388%7C0%2Csh%7C6393%7C30%2Csh%7C6406%7C210&cmenu=null&ppd=0&ppl=0&fbe=&xmv=0&xms=0&xmlc=0&jsfw=jquery&jsfwv=jquery-1.7.1&al=men&scr=0&scv=0&apiu=0&ba=1&sid=5914469ecf915816&rev=v7.13.0-wp&pub=&dp=172.16.44.19&fp=oc%2Findex.php%3Froute%3Dproduct%2Fproduct%26manufacturer_id%3D8%26product_id%3D36&pfm=1&icns=", "Referer=http://172.16.44.19/oc/index.php?route=product/"
		"product&manufacturer_id=8&product_id=36", ENDITEM, 
		"Url=https://ieonline.microsoft.com/iedomainsuggestions/ie11/suggestions.en-US", "Referer=", ENDITEM, 
		"Url=https://ieonline.microsoft.com/ieflipahead/ie10/rules.xml?mkt=en-US", "Referer=", ENDITEM, 
		LAST);
//-------------------------------------------------------------

//Client adds product to cart

	web_submit_data("Add to cart", 
		"Action=http://172.16.44.19/oc/index.php?route=checkout/cart/add", 
		"Method=POST", 
		"RecContentType=text/html", 
		"Referer=http://172.16.44.19/oc/index.php?route=account/wishlist", 
		"Snapshot=t13.inf", 
		"Mode=HTML", 
		ITEMDATA, 
		"Name=product_id", "Value=36", ENDITEM, 
		"Name=quantity", "Value=1", ENDITEM, 
		LAST);
//-------------------------------------------------------------

//Client moves to his account

	web_reg_find("Text=My Account", 
		LAST);

	lr_think_time(4);
	
	web_url("From wish list to my account", 
		"URL=http://172.16.44.19/oc/index.php?route=account/account", 
		"Resource=0", 
		"RecContentType=text/html", 
		"Referer=http://172.16.44.19/oc/index.php?route=account/wishlist", 
		"Snapshot=t15.inf", 
		"Mode=HTML", 
		LAST);
//-------------------------------------------------------------

//Client moves to his shopping cart

	web_reg_find("Text=Shopping Cart", 
		LAST);

	web_link("View Cart", 
		"Text=View Cart", 
		"Snapshot=t16.inf", 
		LAST);	
//-------------------------------------------------------------

//Client removes product from his shopping cart

	lr_think_time(4);
	
	web_image("Remove", 
		"Alt=Remove", 
		"Snapshot=t17.inf", 
		LAST);
//-------------------------------------------------------------

//Client moves to home page
	web_reg_find("Text=Your Store", 
		LAST);

	web_link("Home Page", 
		"Text=Home", 
		"Ordinal=1", 
		"Snapshot=t18.inf", 
		LAST);
//-------------------------------------------------------------

//Client opens his wish list

	web_reg_find("Text=My Wish List", 
		LAST);

	lr_think_time(4);
	
	web_link("Wish List (1)", 
		"Text=Wish List (1)", 
		"Snapshot=t19.inf", 
		LAST);
//-------------------------------------------------------------

//Client removes product from his wish list

	web_image("Remove_2", 
		"Alt=Remove", 
		"Snapshot=t20.inf", 
		LAST);
//-------------------------------------------------------------

//Client got redirected to his account

	web_reg_find("Text=My Account", 
		LAST);

	web_link("test1", 
		"Text=test1", 
		"Snapshot=t21.inf", 
		LAST);
//-------------------------------------------------------------

//Clien opens home page

	web_reg_find("Text=Your Store", 
		LAST);

	lr_think_time(5);

	web_link("Home_3", 
		"Text=Home", 
		"Ordinal=1", 
		"Snapshot=t22.inf", 
		LAST);
//-------------------------------------------------------------
	

//Client logs out
	
	web_reg_find("Text=Account Logout", 
		LAST);
	 
       lr_start_transaction("Logout");

	web_link("Logout", 
		"Text=Logout", 
		"Snapshot=t23.inf", 
		LAST);

       lr_end_transaction("Logout", LR_AUTO);
       
//-------------------------------------------------------------
       
//Client moves to home page

	web_reg_find("Text=Your Store", 
		LAST);

	web_link("Home_4", 
		"Text=Home", 
		"Ordinal=1", 
		"Snapshot=t24.inf", 
		LAST);
//-------------------------------------------------------------

	return 0;
}